.byebug_history

```diff
@@ -1,3 +1,15 @@
+exit
+c
+args
+args.detect { |arg| arg.is_a? Symbol }
+args.detect { |arg| arg.is_a? Symbol }.map(&:to_s)
+args.detect { |arg| arg.is_a? Symbol }.join ' '
+args.detect { |arg| arg.is_a? Symbol }
+args
+c
+argument.map { |k, v| "#{k}: #{get_arg_value_str(v)}" }
+argument
+args
 c
 next
 nex
```

Comment:
Did you mean to check this one in? Maybe `git rm`?

Reply:
i thought i removed this! but maybe on different branch? will remove it in this branch now.